backpack/extensions/firstorder/variance/rnn.py

```diff
@@ -0,0 +1,20 @@
+"""Contains VarianceRNN."""
+from backpack.extensions.firstorder.gradient.rnn import GradRNN
+from backpack.extensions.firstorder.sum_grad_squared.rnn import SGSRNN
+from backpack.extensions.firstorder.variance.variance_base import VarianceBaseModule
+
+
+class VarianceRNN(VarianceBaseModule):
+    """Extension for RNN, calculating variance."""
+
+    def __init__(self):
+        """Initialization."""
+        super(VarianceRNN, self).__init__(
+            params=["bias_ih_l0", "bias_hh_l0", "weight_ih_l0", "weight_hh_l0"],
+            grad_extension=GradRNN(),
+            sgs_extension=SGSRNN(),
+        )
+
+    @staticmethod
+    def _get_axis_batch() -> int:
+        return 1
```

Comment:
I'm going to run into trouble with this solution. Because as soon as `batch_first = True` for RNNs is supported in BackPACK I need to access `module.batch_first` but there is no access to `module`. Would leave it for now.

Reply:
Good point. Marking as resolved for now.